provider/cmd/manifest.go

```diff
@@ -101,10 +103,12 @@ func doSendManifest(cmd *cobra.Command, sdlpath string) error {
 		res := result{
 			Provider: prov,
 			Status:   "PASS",
-			Error:    err,
 		}
-
 		if err != nil {
+			res.Error = err.Error()
+			if e, valid := err.(gwrest.ClientResponseError); valid {
```

Comment:
Isn't this a use case for `errors.As` ?

Reply:
`errors.As` returns bool only, still need to make type assertion to get Message from it